### PR TITLE
docs: Present Proof Acknowledgement message

### LINKED ISSUE #40: [DIDComm] Success/Redirection after Wallet Presentation Submission
In Request/Share section, the [wallet receives an empty success response or a redirectURL](https://identity.foundation/waci-presentation-exchange/#response) when verifier validates the presentation_submission. 

But [in case of DIDComm](https://identity.foundation/waci-presentation-exchange/#message-2-present-proof), how would the wallet knows that the submission was successful or the redirectURL ? DIDComm [being transmit only](https://identity.foundation/didcomm-messaging/spec/#https), the http 202 indicates that the other agent has received the message and not essentially processed/validated it.

One solution would be to wait for [problem-report](https://github.com/hyperledger/aries-rfcs/blob/808daf80b7dbf084207a6c1627ce351247c90b78/features/0035-report-problem/README.md) from verifier with some timeout in wallet (not an ideal solution).

### PR DESCRIPTION
**Pull request by rolsonquadras**
Add DIDComm present-proof ack message to confirm the successful presentation validation by the verifier to prover

closes #40

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -846,10 +846,37 @@ For context on the Vaccination object passed, see the W3C-CCG [Vaccination
 Vocabulary](https://w3c-ccg.github.io/vaccination-vocab/), from which the
 example is drawn.
 
+### Message 4 - Ack Presentation
+
+Once the verifier validates the presentation, it can send a
```
Comment (kdenhartog) [inline spec/spec.md:851]:
```suggestion
Once the verifier validates the presentation, it MAY send a
```

**Review by kdenhartog — CHANGES_REQUESTED** [review]
Indicates that not all verifiers will implement this feature in a normative way. If we expect all verifiers MUST send this message we should update to a MUST.

File: spec/spec.md (hunk 1 above)
Comment (rolsonquadras) [reply spec/spec.md]: Thanks, [updated](https://github.com/decentralized-identity/waci-presentation-exchange/compare/89dacf6f4d58736ab339e3b5c79e354f6f0383ab..0430d3ba7b4f0999389244d0c9347742ab2666b6) the text.

**Review by kdenhartog — APPROVED** [review]

**Review by OR13 — APPROVED** [review]
As someone not familiar with DIDComm this looks good to me.

File: spec/spec.md [hunk 2]
```diff
@@ -846,10 +846,37 @@ For context on the Vaccination object passed, see the W3C-CCG [Vaccination
 Vocabulary](https://w3c-ccg.github.io/vaccination-vocab/), from which the
 example is drawn.
 
+### Message 4 - Ack Presentation
```
Comment (OR13) [inline spec/spec.md:849]:
```suggestion
### Step 5 - Ack Presentation
```

Comment (OR13) [reply spec/spec.md]: We agreed to do this on the call.

**Comment by OR13** [comment]
We agreed to merge this on the call.

**Pull request merged**